src/main/java/com/gooddata/warehouse/Warehouse.java

```diff
@@ -143,21 +151,23 @@ public void setEnvironment(final Environment environment) {
         setEnvironment(environment.name());
     }
 
+    public void setConnectionUrl(final String connectionUrl) {
```

Comment:
It needs to be set in the HDS REST API, since there is no other way of getting the connection URL into the object.